modules/build/src/main/scala/scala/build/preprocessing/ScalaPreprocessor.scala

```diff
@@ -60,7 +60,7 @@ case object ScalaPreprocessor extends Preprocessor {
     UsingPublishDirectiveHandler
   )
 
-  val requireDirectiveHandlers = Seq[RequireDirectiveHandler](
+  val requireDirectiveHandlers: Seq[RequireDirectiveHandler] = Seq[RequireDirectiveHandler](
```

Comment:
Do we really need this? There's already a `Seq[RequireDirectiveHandler]` on the same line…

Reply:
right, no need to pass the type param twice, in the same line, my bad.
should be better now.